README.md

```diff
@@ -21,9 +21,8 @@ Simple build instructions: use `make` to build. This will build the customisatio
 to your `PATH` or copy it to somewhere in your `PATH` eg `sudo cp bin/moby /usr/local/bin/`. Or you can use `sudo make install`.
 
 If you already have `go` installed you can use `go get -u github.com/docker/moby/src/cmd/moby` to install
-the `moby` tool, and then use `moby build linuxkit.yml` to build the example configuration. You
-can use `go get -u github.com/docker/moby/src/cmd/infrakit-instance-hyperkit` to get the
-hyperkit infrakit tool.
+the `moby` tool. Youcan use `go get -u github.com/docker/moby/src/cmd/infrakit-instance-hyperkit`
```

Comment:
same here - could we change this command to go get -u github.com/linuxkit/linuxkit/src/cmd/infrakit-instance-hyperkit?